sqlx-core/src/pool/options.rs

```diff
@@ -84,6 +83,27 @@ pub struct PoolOptions<DB: Database> {
     pub(crate) parent_pool: Option<Pool<DB>>,
 }
 
+// Manually implement `Clone` to avoid a trait bound issue.
+//
+// See: https://github.com/launchbadge/sqlx/issues/2548
+impl<DB: Database> Clone for PoolOptions<DB> {
+    fn clone(&self) -> Self {
+        PoolOptions {
+            test_before_acquire: self.test_before_acquire,
+            after_connect: self.after_connect.as_ref().map(Arc::clone),
```

Comment:
`Option` actually implements `Clone` which forwards to the inner type: https://doc.rust-lang.org/stable/std/option/enum.Option.html#impl-Clone-for-Option%3CT%3E

So you can change all of these to just `.clone()`